test/e2e/storage/testsuites/volumemode.go

```diff
@@ -196,13 +196,33 @@ func (t *volumeModeTestSuite) defineTests(driver TestDriver, pattern testpattern
 				framework.ExpectNoError(framework.WaitOnPVandPVC(l.cs, l.ns.Name, l.pv, l.pvc))
 
 				ginkgo.By("Creating pod")
-				pod, err := framework.CreateSecPodWithNodeSelection(l.cs, l.ns.Name, []*v1.PersistentVolumeClaim{l.pvc},
-					nil, false, "", false, false, framework.SELinuxLabel,
-					nil, framework.NodeSelection{Name: l.config.ClientNodeName}, framework.PodStartTimeout)
+				pod := framework.MakeSecPod(l.ns.Name, []*v1.PersistentVolumeClaim{l.pvc}, nil, false, "", false, false, framework.SELinuxLabel, nil)
+				// Setting node
+				pod.Spec.NodeName = l.config.ClientNodeName
+				pod, err = l.cs.CoreV1().Pods(l.ns.Name).Create(pod)
 				defer func() {
 					framework.ExpectNoError(framework.DeletePodWithWait(f, l.cs, pod))
 				}()
-				framework.ExpectError(err)
+				framework.ExpectNoError(err)
```

Comment:
should have an annotation (e.g. Failed to create...)

Reply:
Fixed all in volumemode.go